gnucash/import-export/import-main-matcher.c

```diff
@@ -1100,10 +1105,16 @@ gnc_gen_trans_edit_fields (GtkMenuItem *menuitem, GNCImportMainMatcher *info)
                                     DOWNLOADED_COL_DESCRIPTION_STYLE, style,
                                     -1);
                 xaccTransSetDescription (row->trans, new_desc);
+                info->edits = g_list_prepend (info->edits, (gpointer)CACHE_INSERT (new_desc));
```

Comment:
I don't think the string cache adds any value here vs. just `strdup`ing the string.